lib/Doctrine/Common/Annotations/DocParser.php

```diff
@@ -988,18 +988,7 @@ private function Identifier()
 
         $this->lexer->moveNext();
 
-        $className = $this->lexer->token['value'];
-
-        while ($this->lexer->lookahead['position'] === ($this->lexer->token['position'] + strlen($this->lexer->token['value']))
```

Comment:
because lookahead can be null here, this line triggers a deprecation on 7.4
that's one of the main remaining blockers we have to make Symfony's tests pass on PHP 7.4 :)

Reply:
@guilhermeblanco let's separate the concern of fixing 7.4 compat and the issue found with this patch: we should check for the lookahead to exist in this `while()`, and have a test that triggers a deprecation warning on 7.4